tools/add_triage_signature.py

```diff
@@ -32,6 +32,7 @@
 CF_DOMAIN = "12319045"
 CF_CLUSTER_ID = "12316349"
 CF_FUNCTION_IMPACT = "12317358"
+CF_IGNORED_DOMAINS = ["redhat.com", "juniper.net"]
```

Comment:
should be { } set not [ ] list

Reply:
(nit, of-course. it's insanely negligible performance-wise, it's just makes the intent more clear)